lib/shared/units.js

```diff
@@ -50,33 +43,38 @@ const MEGABYTE_TO_BYTE_RATIO = 1e+6;
 const MILLISECONDS_IN_A_DAY = 86400000;
 
 /**
- * @summary Convert bytes to gigabytes
+ * @summary Convert bytes to megabytes
  * @function
  * @public
  *
  * @param {Number} bytes - bytes
- * @returns {Number} gigabytes
+ * @returns {Number} megabytes
  *
  * @example
- * const result = units.bytesToGigabytes(7801405440);
+ * const result = units.bytesToMegabytes(7801405440);
  */
-exports.bytesToGigabytes = (bytes) => {
-  return bytes / GIGABYTE_TO_BYTE_RATIO;
+exports.bytesToMegabytes = (bytes) => {
```

Comment:
AFAICT (but I could be wrong), that speed only needs to be a number, because of https://github.com/resin-io/etcher/blob/master/lib/gui/pages/main/templates/main.tpl.html#L98
So could we instead change it to just be a string (created by pretty-bytes) instead, and get rid of the formatting and 'MB' in `main.tpl.html` ? This would also mean that if somebody had a really slow drive, Etcher might display "123 KB/s" instead of "0.12 MB/s", which might be nicer?